### ISSUE TITLE
Open climate data seen by mountain glaciers as a natural low-pass filter

### ISSUE THREAD

**Issue by fmaussion** [OpenDataHack2018/OpenDataHack2018-Brainstorming#9]
**Problem**

Glaciers constitute natural low-pass filters of atmospheric variability. They allow people to directly perceive slow changes of the climate system, that would otherwise be superimposed by short-term noise in human perception. Since glaciers form prominent features of many landscapes, shrinking glaciers have become an icon of climate change.

There are very few large-scale glacier models available to date, and only one of them is open-source. An interactive visualization tool displaying near real-time global current (and projected) glacier change would have a great value for public awareness and education about glaciers and climate change. 

**Proposed solution**

In this project I propose to drive the [Open Global Glacier Model](http://oggm.org) (OGGM) with all major gridded climate datasets (present & future) and analyze them under a "glacier perspective". Differences between projections will be made visible thanks to interactive visualizations (regional maps, timeseries, single glaciers) and a companion website will explain the model output for a wider audience.

**Topics** 

Hack-Vis, Hack-Out

**Data**

Monthly averages of gridded temperature and precipitation (e.g. from GCMs, reanalyses or gridded observations). The longer the dataset the better it is.

**Team**

- Fabien Maussion - role: interface glacier model / climate data, (project management) 

**Is your team incomplete and you need extra help?**

Yes I would definitely need help! I am a scientist with a reasonable programming background, but I would be glad to be joined by:

- Web developer / visualization expert with experience in dashboards (e.g. [Dash](https://plot.ly/products/dash/) or [Bokeh](https://bokeh.pydata.org))
- Software developer
- Project manager
- Science - general public communication expert
- ...

**Comment by fmaussion** [comment]
@OpenDataHack2018 , after a quick look at the CDS catalogue: there doesn't seem to be any CMIP or long-term reanalysis data available (yet). Will any of these long-term (> 30 years) gridded datasets be available before the event?

**Comment by fmaussion** [comment]
Currently the only gridded climate data available are ERA-I and ERA-5. This reduces the number of possible visualizations but I suggest to keep the project open and provide a "template" instead of full visualizations. Let me know if that sounds interesting to you!

**Comment by kaedonkers** [comment]
It sounds interesting to me! Would a Python programmer be useful to you?

**Comment by fmaussion** [comment]
@kaedonkers absolutely! According to the agenda there will be some time in the morning to present the projects, I'm happy to tell you more about it.

**Comment by fmaussion** [comment]
Small update and a side project: in order to leverage the CDS ERA5 and glacier data, a great visualization would be to explore the location of all glaciers worldwide and visualize climate statistics where there are, i.e find out under which type of climate glaciers are found: temperature, solid precipitaiton, continentality, dry/wet, etc...

**Comment by OpenDataHack2018** [comment]
This challenge was selected by team: Glaciers

**Comment by fmaussion** [comment]
(Close to) Final result:  https://dash.klima.uni-bremen.de/
